jscomp/runtime/caml_format.js

```diff
@@ -608,122 +606,105 @@ function caml_format_int(fmt, i) {
 
 function caml_int64_format(fmt, x) {
   var f = parse_format(fmt);
-  var x$1 = f[/* signedconv */5] && x[/* hi */1] < 0 ? (f[/* sign */8] = -1, Caml_int64.neg(x)) : x;
+  var x$1 = f[/* signedconv */5] && Caml_int64.lt(x, /* int64 */[
+        0,
+        0
+      ]) ? (f[/* sign */8] = -1, Caml_int64.neg(x)) : x;
   var s = "";
   var match = f[/* base */4];
   switch (match) {
     case 0 : 
-        var wbase = /* record */[
+        var wbase = /* int64 */[
```

Comment:
shall we cache some small integers like `8L`, `0L`, `10L` to avoid allocation each time...